### PR TITLE
Feature/nsx product card overview

### PR DESCRIPTION
**Pull request by MekalaNagarajan-Centrica**

### CODE REVIEW

File: docs/components/nsx-product-card-overview.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+description: Experience | Product Card Comparison.
+---
+
+## Introduction
+
+The product card overview is an experience pattern that can be used for product comparison with excess selection option to show different HomeCare products and its prices.
```
Comment (PtitBen) [inline docs/components/nsx-product-card-overview.md:7]: How about this?

> 
The product card overvie is an experience pattern that can be used to compare HomeCare products against each other and see the difference in price when choosing to have an excess or not without having to reload the entire page.

File: docs/components/nsx-product-card-overview.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Content guidance
+
+### With Excess
+When `excess` property of the overview component is set with array of price options, the excess prices are displayed as radio options in the overview component. When user selects an radio option, `excess-change` event is dispatched from the overview component. The consuming wrapper component has to listen for the event and set the updated ns-product-card prices back in `data` attribute of the component.
```
Comment (PtitBen) [inline docs/components/nsx-product-card-overview.md:12]: Do we need to add "an" before array?
> When `excess` property of the overview component is set with **an** array of price options

--

I feel we need "a" before user. And the an before radio should be a 😊 
> When **a** user selects **a** radio option

File: docs/components/nsx-product-card-overview.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+<nsx-product-card-overview data='[{"monthly": 1200,"annually": 14400,"excess": 90}]' priceheading="Estimated Price"></nsx-product-card-overview>
+```
+![Experience pattern - Product Card Comparison](https://user-images.githubusercontent.com/78729952/110448202-e177e180-80b8-11eb-8a02-6c40db07a9ba.png)
+
+## Specification
+
+| Attribute | Type | Default | Options | Description |
+| :--- | :--- | :--- | :--- | :------ |
+| `data` | `array` | `[]` | `required` | The list of price values to be updated in the product card price slots |
+| `excess` | `array` | `[]` | `optional` | The list of excess price that will be displayed to the user for excess selection|
```
Comment (PtitBen) [inline docs/components/nsx-product-card-overview.md:35]: Need an s to prices

> The list of excess price**s**

File: docs/components/nsx-product-card-overview.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+| `heading` | `h` tag |
+| `content` | `ns-content` tag |
+| `product-*` |  `ns-product-card` tag |
+| `download-files` | `ul` tag with nested multiple`li` of `ns-download` for multiple files |
+
+## Events
+
+| Event | Description | Details |
+| :--- | :-----| :--- |
+| `excess-change` | Dispatched when the user change the excess price radio option | |
```
Comment (PtitBen) [inline docs/components/nsx-product-card-overview.md:49]: need an "s" to changes

> Dispatched when the user change**s**

**Review by PtitBen — COMMENTED** [review]
**What decoration can be used?**:  invert-concave-grey-light


Could we hve this information maybe in the specification, or even in the content guidance or usage as a sentence. It can be very easily missed. 

And because we only allow one option for the decoration (when usually there are plenty decorations to choose) it's nice to explain that, and why only one choice.

File: docs/components/nsx-product-card-overview.md (hunk 1 above)
Comment (RobTobias123) [reply docs/components/nsx-product-card-overview.md:7]: May I suggest a slight variation of Ben's:

The product card overview is an experience pattern that can be used to compare HomeCare products against each other and see the difference in price when choosing to have an excess or not  – without having to reload the entire page.

Comment (PtitBen) [reply docs/components/nsx-product-card-overview.md:7]: Wow, if I make typos in my suggestions ... 

Rob's version is great !!!

File: docs/components/nsx-product-card-overview.md (hunk 2 above)
Comment (RobTobias123) [reply docs/components/nsx-product-card-overview.md:12]: Suggestion: 

When an ‘excess’ property of the overview component is set with an array of price options, the excess prices are displayed as radio options in the overview component. When a user selects a radio option, an ‘excess-change’ event is dispatched from the overview component. The consuming wrapper component has to listen for the event and sets the updated ns-product-card prices back in the ‘data’ attribute of the component.

**Review by PtitBen — APPROVED** [review]
Looks good to me !

**Review by RobTobias123 — COMMENTED** [review]
Typo line 10 - 'recommanded'  - 'recommend'?

**Review by RobTobias123 — APPROVED** [review]
Looks good to me!

**Comment by MekalaNagarajan-Centrica** [comment]
Decoration recommendation is updated in documentation (https://github.com/ConnectedHomes/nucleus/issues/1930)

**Pull request merged**